### ISSUE TITLE
wk2 april_epic Story K&L, Story B: Health Equity Grant & Microsoft Dynamics - Quantify the scale of emails in mtl.help and mtl.info #crm #365

### ISSUE THREAD

**Issue by epearman** [lzim/teampsd#2900]
#health_equity_grant
#pivot
#CRM

- [ ] @epearman and @emilymetcalf114 to investigate quantifying the scale of emails in mtl.help and mtl.info in Microsoft dynamics.

- [ ] mtl.help@va.gov emails by month 
- [ ] mtl.info@va.gov emails by month

**Comment by epearman** [comment]
**wk2 2023_apr_epic msd365_workflow** 
@emilymetcalf114  #people_ops_workstream
@epearman #data_ops_workstream

- [ ] Reviewed section **9.2** Run, create, edit and share Reports in the [Dynamics Testers Guide](https://dvagov.sharepoint.com/:w:/r/sites/teampsd_vha/Shared%20Documents/msd365_workgroup/Dynamics%20365/Modeling%20to%20Learn%20D365%20Testers%20Guide.docx?d=w86160ba5093e4588930a492c74090aa8&csf=1&web=1&e=uqbKyA)
- [ ] Tested viewing of existing and creation of new reports in the [Dynamics DEV Environment ](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdvagov-psd-dev.crm9.dynamics.com%2F&data=04%7C01%7C%7C4b6acaaeed0642fbbd1508da18daaf46%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C637849624764318511%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=vdcPJHJ9kCKBW%2B1IM5TpX51BzuNzzHzYYeCos0G8hog%3D&reserved=0)

**Discoveries**
- [ ] Can create reports to filter and group on specific variables and views 
- [ ] Can only view existing reports if related activities and teams being reported on have been assigned to the Dynamics user. If nothing is assigned to the user the report will be blank.  
    - [ ] Example need to assign activities to user before you can view the "Email by Team" report. 
![image](https://user-images.githubusercontent.com/67978478/230969405-e6d0365b-0c3a-491c-b289-a8cd3cb889df.png)

![image](https://user-images.githubusercontent.com/67978478/230969623-c8feb340-0159-403d-8159-1f906b3c9746.png)


**Functional Requirements**
- [ ]

**Comment by epearman** [comment]
Closing April efforts on this car. Will open another card on this for May. This effort is currently blocked by bug #2913
